app/assets/javascripts/ama_layout/forms/placeholder.coffee

```diff
@@ -0,0 +1,30 @@
+class AMALayout.Placeholder
+  constructor: (args = {}) ->
+    args = $.extend({}, @defaultVariables(), args)
+    @placeholderTextColor = args.placeholderTextColor
+    @inputTextColor = args.inputTextColor
+    @defaultTextList = args.defaultTextList
+    @setup()
+
+  setup: ->
+    for defaultText in @defaultTextList
+      @changeDefaultPlaceholderColor(defaultText)
+      @updatePlaceholderColor(defaultText)
+
+  changeDefaultPlaceholderColor: (defaultText) ->
+    $("option:selected:contains('#{defaultText}')").parent('select').css('color', @placeholderTextColor)
```

Comment:
We tried that, it didn't work because it's the option that contains the text, but the select that needs to change colour.